include/cantera/transport/GasTransport.h

```diff
@@ -117,17 +117,43 @@ class GasTransport : public Transport
 
     //! Return the polynomial fits to the viscosity of species i
     //! @see fitProperties()
-    virtual void getViscosityPolynomials(size_t i, double* coeffs) const;
+    virtual void getViscosityPolynomial(size_t i, double* coeffs) const;
 
     //! Return the temperature fits of the heat conductivity of species i
     //! @see fitProperties()
-    virtual void getConductivityPolynomials(size_t i, double* coeffs) const;
+    virtual void getConductivityPolynomial(size_t i, double* coeffs) const;
 
     //! Return the polynomial fits to the binary diffusivity of species pair (i, j)
     //! @see fitDiffCoeffs()
-    virtual void getBinDiffusivityPolynomials(size_t i, size_t j, double* coeffs) const;
+    virtual void getBinDiffusivityPolynomial(size_t i, size_t j, double* coeffs) const;
+
+    //! Return the polynomial fits to the collision integral of species pair (i, j)
+    //! @see fitCollisionIntegrals()
+    virtual void getCollisionIntegralPolynomial(size_t i, size_t j, double* astar_coeffs, 
```

Comment:
Now that all functions are first defined in `TransportBase.h` (with a specialization defined here), the docstrings should be moved to the base and can be omitted here. 

Same for all other newly introduced methods below.

Reply:
I added the docstrings in `TransportBase.h` but I will keep them in `GasTransport.h` as well, to have a link (@ see) to `fitProperties()`, `fitDiffCoeffs()` and `fitCollisionIntegrals()`, which are only defined in `GasTransport.h`